src/sqswatcher/sqswatcher.py

```diff
@@ -123,7 +123,9 @@ def _get_sqs_queue(region, queue_name, proxy_config):
     :return: the Queue object
     """
     log.debug("Getting SQS queue '%s'", queue_name)
-    sqs = boto3.resource("sqs", region_name=region, config=proxy_config)
+    sqs = boto3.resource(
+        "sqs", region_name=region, config=proxy_config, endpoint_url="https://sqs." + region + ".amazonaws.com"
```

Comment:
Same here for cn region. Just pasting here the utility function we use for STS in the CLI code in case you want to follow the same approach:

```
def _get_sts_endpoint():
    """Get regionalized STS endpoint."""
    region = get_region()
    return "https://sts.{0}.{1}".format(region, "amazonaws.com.cn" if region.startswith("cn-") else "amazonaws.com")
```